text-minimessage/src/main/java/net/kyori/adventure/text/minimessage/tag/standard/StandardTags.java

```diff
@@ -288,6 +289,17 @@ public static TagResolver transition() {
     return ShadowColorTag.RESOLVER;
   }
 
+  /**
+   * Get a resolver for the {@value SpriteTag#SPRITE} tag.
+   *
+   * @return a resolver for the {@value SpriteTag#SPRITE} tag.
+   * @since 4.25.0
+   * @sinceMinecraft 1.21.9
+   */
+  public static @NotNull TagResolver spriteTag() {
```

Comment:
```suggestion
  public static @NotNull TagResolver sprite() {
```
this should probably match the other tag method names